code/lib/cli/src/generators/REACT_SCRIPTS/index.ts

```diff
@@ -38,8 +39,8 @@ const generator: Generator = async (packageManager, npmOptions, options) => {
     extraPackages.push('prop-types');
   }
 
-  // preset v3 is compat with older versions of CRA, otherwise let version float
-  const extraAddons = [`@storybook/preset-create-react-app${isCra5 ? '' : '@3'}`];
+  const version = versions['@storybook/preset-create-react-app'];
```

Comment:
Is this version compatible with all versions of CRA?  Or are we only supporting the latest CRA now?  If so, should we throw an error or something if a user tries to add SB 7 to an old CRA app?

Reply:
I can throw an error is the version is below v5, yes. I think that's the right thing to do.
 
 I don't want to do active support for CRA <5 and storybook 7>